### ISSUE TITLE
page.goto response issue

### ISSUE THREAD

**Issue by UCSBGauchos** [puppeteer/puppeteer#1283]
Hi

I use the page.goto(fileurl) to download the file. 

Is there any possibility for the page.goto(fileurl) to return the downloaded filename in the response's header. 

```
    for (let item of urlList) {
        console.dir(item)
        const response = await page.goto(item, {
            timeout: 0
          });
        console.dir(await response.headers)
    }
```

**Comment by UCSBGauchos** [comment]
It looks like there's no content-disposition fields in the headers. Can I enable it ? 
{ status: '200',
  date: 'Sun, 05 Nov 2017 05:29:05 GMT',
  'content-encoding': 'gzip',
  'set-cookie': 'AWSALB=itD62msHw5PTYOaChzT+W3Do32uN81520fT2uO9gveLNRVsjytmkLPe47Jehw9WDfM7cat1yWE8C5yL8cohUYBQRhcfC1dua0hhqvlQ++VX6Sx/LU9HqlmTPui/o; Expires=Sun, 12 Nov 2017 05:29:04 GMT; Path=/',
  vary: 'Accept-Encoding,User-Agent',
  'content-type': 'text/html;charset=UTF-8' }

**Comment by aslushnikov** [comment]
> Is there any possibility for the page.goto(fileurl) to return the downloaded filename in the response's header.

I'm not aware of any, but hopefully this will be solved with #299.
